File: CS/ModuleX/VarsModule.cs
// VBConversions Note: VB project level imports
using System.Data;
using System.Collections.Generic;
using PDSA.Data;
using PDSA.WinForms.Forms;
using PDSA.Security;
using System.Diagnostics;
using PDSA.Common;
using PDSA.WinForms;
using PDSA.WinForms.Security;
using PDSA.PDSACryptography;
using DoorsPOS.Utility;
using Microsoft.VisualBasic;
using System.Collections;
using System;
using DoorsPOS.BusinessLayer;
using System.Windows.Forms;
using System.Drawing;
using System.Linq;
using DoorsPOS.DataLayer;
// End of VB project level imports


namespace DoorsPOS.WinApp
{
	sealed class VarsModule
	{
		public static decimal TotalSold = 0;
		public static int QtySold = 0;
		public static int QtyRefund = 0;
		public static bool payconfirm = false;
		public static decimal vpTotalSales = 0;
		public static int vEmpID = 0;
		
		public static void OpenCashdrawer()
		{
			//Modify DrawerCode to your receipt printer open drawer code
			string DrawerCode = '\u001B' + "\u0070" + "\u0030" + "\u0040" + "\u0040";
			//Modify PrinterName to your receipt printer name
			string PrinterName = "EPSON TMU220";
			
			mdlPrint.RawPrinter.PrintRaw(PrinterName, DrawerCode);
		}
	}
	
}

File: DoorsPOS.WinApp/ModuleX/VarsModule.vb
Module VarsModule
    Public TotalSold As Decimal = 0
    Public QtySold As Integer = 0
    Public QtyRefund As Integer = 0
    Public payconfirm As Boolean = False
    Public vpTotalSales As Decimal = 0
    Public vEmpID As Integer = 0
    Public vSalesNum As Integer = 0

    Public Sub OpenCashdrawer()
        'Modify DrawerCode to your receipt printer open drawer code
        Dim DrawerCode As String = Chr(27) & Chr(112) & Chr(48) & Chr(64) & Chr(64)
        'Modify PrinterName to your receipt printer name
        Dim PrinterName As String = "EPSON TMU220"

        RawPrinter.PrintRaw(PrinterName, DrawerCode)
    End Sub
End Module
